quick_tour/flex_recipes.rst

```diff
@@ -194,27 +194,19 @@ rich API for a ``product`` table? Create a ``Product`` entity and give it the
     use ApiPlatform\Core\Annotation\ApiResource;
     use Doctrine\ORM\Mapping as ORM;
 
-    /**
-     * @ORM\Entity()
-     * @ApiResource()
-     */
+    #[ORM\Entity]
+    #[ApiResource]
```

Comment:
If I don't miss anything, the`ApiResource` annotation cannot be used as an attribute.

Reply:
[It is now possible](https://github.com/api-platform/core/blob/2.6/src/Annotation/ApiResource.php#L75) since ApiPlatform v2.6!